lua/jslog/options.lua

```diff
@@ -22,3 +22,64 @@ vim.opt.colorcolumn = '50,80,100'
 vim.opt.cursorcolumn = true
 
 vim.opt.updatetime = 50
+
+-- Custom clipboard for sharing with host from container context
+local isDevContainer = os.getenv("ISDEVCONTAINER")
+if isDevContainer == "true" then
+    -- The following clipboard is for use with an editor
+    -- running in a container and coupled with a strategy
+    -- which uses the $CLIPBOARDPATH as an intermediary.
+    -- This config specifically relies on a socat emitter
```

Comment:
The config is a little more agnostic that it was before now, so update this comment.

Reply:
Done